privacyidea/api/lib/prepolicy.py

```diff
@@ -1243,7 +1243,9 @@ def is_remote_user_allowed(req, write_to_audit_log=True):
         # there should be only one action value here
         if ruser_active:
             if list(ruser_active)[0] == REMOTE_USER.ACTIVE:
-                res = True
+                res = 1
+            if list(ruser_active)[0] == REMOTE_USER.FORCE:
+                res = 2
```

Comment:
why not return the action string itself? It makes things more clear in my opinion.
We could rename the function to `remote_user_allowed` and just check in the two places where it is called if the result matches the strings (or use some kind of enum).

Reply:
Sounds good to me. I will look into this.

Reply:
Done in ae99a827